src/Plugin/MediaEmbedDialog/Image.php

```diff
@@ -80,7 +80,7 @@ public function isApplicable() {
    *
    * @see Linkit::processLinkitAutocomplete()
    */
-  public static function processLinkitAutocomplete(&$element, FormStateInterface $form_state, &$complete_form) {
```

Comment:
I suspect that no one is using this static function outside of this module but it is something to think about when versioning.